src/components/header/header.tsx

```diff
@@ -171,35 +139,10 @@ const Header: React.FC<HeaderProps> = (props) => {
         <HeaderSwoosh />
         {props.siteTitle}
       </SiteTitle>
-      <div>
-        {props.translation
-          ? languages.map((lng) => {
-              return (
-                <StyledLanguageLink
-                  key={lng}
-                  to={language === lng ? '' : props.translation || ''}
-                  language={lng}
-                  selected={language === lng}
-                >
-                  {lng}
-                </StyledLanguageLink>
-              );
-            })
-          : props.showLanguageSwitch &&
-            languages.map((lng) => {
-              return (
-                <LanguageSwitch
-                  key={lng}
-                  onClick={(e) => {
-                    e.preventDefault();
-                    changeLanguage(lng);
-                  }}
-                  selected={language === lng}
-                >
-                  {lng}
-                </LanguageSwitch>
-              );
-            })}
+      <Wrapper>
+        {(props.translation || props.showLanguageSwitch) && (
```

Comment:
If there are no translations, what will the `LanguageSwitch` show?

Reply:
default is `showLanguageSwitch = true` (eg. blog has no language switch)
if a translation is provided, it is chosen as new pathname.